app/gui/view/component-browser/searcher-list-panel/src/lib.rs

```diff
@@ -760,6 +866,26 @@ impl EntryId {
     }
 }
 
+/// The selected part of the component group and the identifier of this group.
+///
+/// Similar to [`component_group::Selected`], but also contains a group id.
+#[allow(missing_docs)]
+#[derive(Copy, Clone, Debug, Eq, Hash, PartialEq)]
+pub enum Selected {
```

Comment:
This is defined in the FRP section - is it part of FRP? (Maybe it is, just asking)

Reply:
Yes, this is a helper type for FRP API.